java/arcs/core/entity/VariableEntityBase.kt

```diff
@@ -23,7 +23,9 @@ open class VariableEntityBase(
     creationTimestamp: Long = UNINITIALIZED_TIMESTAMP,
     expirationTimestamp: Long = UNINITIALIZED_TIMESTAMP,
     isInlineEntity: Boolean = false
-) : EntityBase(entityClassName, schema, entityId, creationTimestamp, expirationTimestamp) {
+) : EntityBase(
+    entityClassName, schema, entityId, creationTimestamp, expirationTimestamp, isInlineEntity
```

Comment:
Optional nit: this would be cleaner with each param on a separate line.

Reply:
Done.